client/src/App.jsx

```diff
@@ -9,9 +10,17 @@ class App extends React.Component {
     };
   }
 
+  componentDidMount() {
+    axios.get('/api/restaurants/?/reviews', {
+      params: {
+        // r_id: 1,
```

Comment:
Is there a reason this is commented out?

Reply:
I haven't coded in the part where we grab the restaurant from the actual localhost:3000 url yet. I did testing from postman to confirm that the server can talk to the database and grab the correct data. I'll complete that part after this pull is merged.

Thanks!